users/danielo515/process_records.h

```diff
@@ -1,35 +1,37 @@
 #pragma once
 #include "quantum.h"
 
-enum custom_keycodes
-{
-    EPRM = SAFE_RANGE, 
+enum custom_keycodes {
+    EPRM = SAFE_RANGE,
```

Comment:
This line isn't actually needed (nor is this macro).  `EEP_RST` handles the same functionality.

Reply:
> This line isn't actually needed (nor is this macro).
Can you expand on that? Why is not longer needed? It used to be required to define new keycodes on the safe range.

Reply:
He means the `EPRM` keycode specifically. You would remove this line and its handling code in `process_record_user()`, and set `RGB_SLD = SAFE_RANGE` instead.

Reply:
Oh, ok, that makes sense. So what it is outdated is the macro that this references to. This I think it is some code that I got when I downloaded the ergodox source from Orix more than a year ago, so it is probably outdated. Thanks for the clarification.

Reply:
Yeah, sorry, the functionally got migrated into quantum.  I need to bug florian about removing this from the Oryx generation. 

But having an eeprom reset key is VERY useful, IMO.